web/src/composebox_typeahead.js

```diff
@@ -1016,6 +1032,10 @@ export function initialize_topic_edit_typeahead(form_field, stream_name, dropup)
     form_field.typeahead(options);
 }
 
+function is_wildcard_mention(mention) {
+    return ["all", "everyone", "stream"].some((wildcard) => mention.includes(wildcard));
```

Comment:
Why is `includes` the right check -- don't we need to check for equality? Will someone with the name `Ballin" trigger this warning?

Reply:
There's a function `full_name_matches_wildcard_mention` in `people.js`; you can do a prep commit to rename that to `is_wildcard_mention` and export it, and then just call that here.

Reply:
> Why is includes the right check -- don't we need to check for equality? Will someone with the name `Ballin" trigger this warning?

Yeah, you're right. This could result in a false positive match. I've made some changes to `full_name_matches_wildcard_mention` to ensure that it checks whether the mention contains a wildcard symbol.

Reply:
I'm considering where we can place the `is_wildcard_mention` function. It seems like a utility function that is not related to either `people.js` or `composebox_typeahead.js`. Is there a designated location for functions like this?

Reply:
`people.js` is probably fine for this for now.